content/docs/buildpacks/language-family-buildpacks/nodejs.md

```diff
@@ -204,6 +208,20 @@ will be chosen based on a priority list which reflects the order of the list
 above with `server.js` being the highest priority and  `index.js` being the
 lowest.
 
+### Using BP_LAUNCHPOINT
+THIS IS NODE START SPECIFIC
```

Comment:
Could you rephrase this/remove? I think the explanation below is sufficient.

Reply:
Yeah that shouldn't be there